pkgs/development/libraries/libmd/default.nix

```diff
@@ -1,14 +1,26 @@
-{ lib, stdenv, fetchurl, autoreconfHook }:
+{ lib, stdenv, fetchurl, fetchpatch, autoreconfHook }:
 
 stdenv.mkDerivation rec {
   pname = "libmd";
   version = "1.0.4";
 
   src = fetchurl {
-    url = "https://archive.hadrons.org/software/${pname}/${pname}-${version}.tar.xz";
+    urls = [
+      "https://archive.hadrons.org/software/libmd/libmd-${version}.tar.xz"
+      "https://libbsd.freedesktop.org/releases/libmd-${version}.tar.xz"
+    ];
     sha256 = "sha256-9RySEELjS+3e3tS3VVdlZVnPWx8kSAM7TB7sEcB+Uw8=";
   };
 
+  patches = lib.optionals stdenv.isDarwin [
```

Comment:
This patch looks okay to apply on all platforms so it might be better to do so to prevent bitrot.